tool/gh_actions/generate_documentation.sh

```diff
@@ -17,6 +17,9 @@ set -euo pipefail
 # The documentation will be placed in the "doc/api" folder.
 output=$(dart doc --validate-links 2>&1 | tee)
 
+# copy and replace the documentation generated to the website
+cp -rf doc/api/ doc/website/
+
```

Comment:
Maybe make a separate script for copy actions, and leave this one only for generating documentation? In this case, you will need to add a new job step to `.github/general.yaml`.